src/containers/SubscriptionsPage/actions.js

```diff
@@ -1,16 +1,39 @@
-import { getSubscriptions } from '../../services/subscriptions';
+import { getSubscriptions } from '../../services/subscriptions'
+import { api } from '../../services/helpers'
 
-export const FETCH_SUBSCRIPTIONS_REQUEST = 'FETCH_SUBSCRIPTIONS_REQUEST';
-export const FETCH_SUBSCRIPTIONS_SUCCESS = 'FETCH_SUBSCRIPTIONS_SUCCESS';
-export const FETCH_SUBSCRIPTIONS_FAILURE = 'FETCH_SUBSCRIPTIONS_FAILURE';
+export const FETCH_SUBSCRIPTIONS_REQUEST = 'FETCH_SUBSCRIPTIONS_REQUEST'
+export const FETCH_SUBSCRIPTIONS_SUCCESS = 'FETCH_SUBSCRIPTIONS_SUCCESS'
+export const FETCH_SUBSCRIPTIONS_FAILURE = 'FETCH_SUBSCRIPTIONS_FAILURE'
 
-export const fetchSubscriptions = (auth) => dispatch => {
-  dispatch({ type: FETCH_SUBSCRIPTIONS_REQUEST });
-  return getSubscriptions(auth)
+export const DELETE_SUBSCRIPTION_REQUEST = 'DELETE_SUBSCRIPTION_REQUEST'
+export const DELETE_SUBSCRIPTION_SUCCESS = 'DELETE_SUBSCRIPTION_SUCCESS'
+export const DELETE_SUBSCRIPTION_FAILURE = 'DELETE_SUBSCRIPTION_FAILURE'
```

Comment:
Not valid per se, but it's a controversial topic anyway. The point is that if you omit semicolons, the compiler will _try_ to insert them for you but only in [certain situations](http://es5.github.io/#x7.9), which could lead to issues like [this](http://stackoverflow.com/questions/16664824/no-semicolon-before-is-causing-error-in-javascript).